src/components/Frame/tests/Frame.test.tsx

```diff
@@ -41,15 +41,10 @@ describe('<Frame />', () => {
   });
 
   it('renders a TrapFocus with a `trapping` prop set to true around the navigation on small screens and showMobileNavigation is true', () => {
-    Object.defineProperty(window, 'innerWidth', {
-      configurable: true,
-      writable: true,
-      value: 500,
-    });
-
     const navigation = <div />;
     const frame = mountWithAppProvider(
       <Frame showMobileNavigation navigation={navigation} />,
+      {mediaQuery: {isNavigationCollapsed: true}},
```

Comment:
This passes the value to `PolarisTestProvider` which passes it to `MediaQueryContext.Provider` to mock `isNavigationCollapsed`